components/mixins/experience-event/experienceevent-contact-request-details.schema.json

```diff
@@ -32,70 +34,65 @@
         "xdm:customerFeedback": {
           "type": "object",
           "properties": {
```

Comment:
No nesting here at all ?

Reply:
Nesting here remains consistent with the other mixins.